tools/e2e/__tests__/common.test.ts

```diff
@@ -46,7 +46,7 @@ describe("listTmpE2EProjects()", () => {
 			.intercept({
 				path: `/client/v4/accounts/${MOCK_CLOUDFLARE_ACCOUNT_ID}/pages/projects`,
 				query: {
-					per_page: 10,
+					per_page: 100,
```

Comment:
is this expected that 100 doesn't match l57-66 and l70?

Reply:
Yes it is fine. The inbound page size can actually be ignored by the endpoint. The page size in the response is the source of truth.